src/components/Header.jsx

```diff
@@ -69,7 +69,9 @@ const Header = ({
       role='navigation'
     >
       <div className='navbar-header'>
-        <Link className='navbar-brand' to='/'>Strims</Link>
+        <Link className='navbar-brand' to='/'>
+          <img src={logo} />
```

Comment:
consider adding an `alt` tag with value `'Strims'` (for screen readers or for when the image fails to load)